lib/MusicBrainz/Server/Data/Medium.pm

```diff
@@ -321,6 +321,11 @@ sub load_related_info {
     $self->c->model('Track')->load_for_mediums(@mediums);
 
     my @tracks = map { $_->all_tracks } @mediums;
+
+    for my $medium (@mediums) {
+        $medium->has_loaded_tracks(1);
```

Comment:
It might make more sense to move this to `$c->model('Track')->load_for_mediums`, since that method is called in a ton of places throughout the codebase and seems to be what actually loads the tracks.